Server/service/networkService.js

```
@@ -105,13 +105,18 @@ class NetworkService {
 	 */
 	async requestHttp(job) {
 		const url = job.data.url;
+		const config = {};
+
+		job.data.secret !== undefined &&
+			(config.headers = { Authorization: `Bearer ${job.data.secret}` });
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)